src/chia_log/log_consumer.py

```diff
@@ -164,20 +168,23 @@ def create_log_consumer_from_config(config: dict) -> Optional[LogConsumer]:
 
     if enabled_consumer == "network_log_consumer":
         if not check_keys(
-            required_keys=["remote_file_path", "remote_host", "remote_user"], config=enabled_consumer_config
+            required_keys=["remote_file_path", "remote_host", "remote_port", "remote_user"],
```

Comment:
Thanks for the contribution!

Perhaps worth making the port optional so we have a bit better backward compatibility with existing config files, otherwise looks good to me! 

@pieterhelsen tagging you here since you had that issue assigned to yourself, please check it out!

Reply:
Agree, would make the port optional in the config and set it explicitly to 22 if the config is not present.